src/documentation/programming-guide.md

```diff
@@ -789,7 +789,7 @@ Side inputs are useful if your `ParDo` needs to inject additional data when proc
 %}
 
 # We can also pass side inputs to a ParDo transform, which will get passed to its process method.
-# The only change is that the first arguments are self and a context, rather than the PCollection element itself.
+# The only change is that the first arguments are self and the PCollection element itself.
```

Comment:
Yeah no idea. How about changing the sentence to "The first two arguments for the process method would be self and element."